Purchases/Public/Purchases.swift

```diff
@@ -1407,12 +1407,13 @@ public extension Purchases {
      *
      * - Returns: An instantiated `Purchases` object that has been set as a singleton.
      */
-    @objc(configureWithAPIKey:appUserID:observerMode:userDefaults:useStoreKit2IfAvailable:)
-    @discardableResult static func configure(withAPIKey apiKey: String,
-                                             appUserID: String?,
-                                             observerMode: Bool,
-                                             userDefaults: UserDefaults?,
-                                             useStoreKit2IfAvailable: Bool) -> Purchases {
+    // todo: uncomment @objc name, remove private keyword, uncomment APITester calls, uncomment PurchasesTests
+//    @objc(configureWithAPIKey:appUserID:observerMode:userDefaults:useStoreKit2IfAvailable:)
+    @discardableResult private static func configure(withAPIKey apiKey: String,
```

Comment:
perhaps we could keep it `internal`? That way we can still use it for testing ourselves by doing `@testable import RevenueCat`

Reply:
ooohhhhh great call!